### DISCUSSION TITLE
Are the Prisma still doesn't support the Clickhouse database through mysql interface?

### DISCUSSION THREAD

**Discussion by pmerkul0v** [prisma/prisma#19177] [Q&A] [answered]
I saw the error in this ticket https://github.com/prisma/prisma/issues/16174. But i have updated the prisma client and got other error. 

campaign = await this.prismaPerformance.campaign_info.findMany(
Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Server(ServerError { code: 48, message: "Code: 48. DB::Exception: Command \u{16} is not implemented. (NOT_IMPLEMENTED) (version 23.4.2.11 (official build))", state: "00000" })), transient: false })

**Comment by ludralph** [comment, marked as answer]
Hi @pmerkul0v 

We currently do not have full support for Clickhouse. The ticket you linked is a feature request for supporting Clickhouse. You can add a 👍 to it to enable our engineers prioritize the feature.

**Reply by pmerkul0v** [reply]
Thank you for the answer!

**Reply by ludralph** [reply]
You're welcome.

**Reply by janpio** [reply]
The error message you shared by the way that Clickhouse does not fully support the MySQL protocol. That is unfortunately what makes it hard for us to support them without major engineering effort to find workarounds.
